app/generators/questionnaire_generator.rb

```diff
@@ -14,7 +14,7 @@ def generate_questionnaire(response_id, content, title, submit_text, action, aut
       body = safe_join([
                          questionnaire_header(title),
                          questionnaire_hidden_fields(response_id, authenticity_token),
-                         questionnaire_questions(content),
+                         questionnaire_questions(content, response_id),
```

Comment:
Ik gebruik response_id in de hashing van de stop_questionnaire_tokens om ervoor te zorgen dat je niet zomaar het stop_questionnaire token van één vragenlijst in een andere vragenlijst kan injecten.

Anders kan het namelijk een probleem zijn wanneer je twee vragenlijsten hebt die identiek zijn behalve dat de ene een stop_subscription property bij een optie heeft. De andere vragenlijst heeft precies dezelfde vragen en opties, maar geen stop_subscription property. Doordat we het response id meehashen is een eventueel geinjecte stop_subscription token dus niet geldig in andere vragenlijsten (letterlijk: in andere responses).